### PR TITLE
Added 'Add' and 'Use'  methods for Startup in ASP.net Core

### PR DESCRIPTION
**Pull request by maikebing**
- Added  Asp.Net Core Sample
- Added 'Add' and 'Use'  methods for Startup in ASP.net Core

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: csharp
 sudo: required  
 dist: trusty  
 mono: none
-dotnet: 2.1
+dotnet: 2.2
```
Comment (mukmyash) [inline .travis.yml]: Travis can't build with this configuration.
For fix this issue, you must set 'xenial' value for property 'dist' (line 3)
 
more info: https://github.com/travis-ci/travis-ci/issues/9684#issuecomment-447683968

**Review by mukmyash — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (mukmyash) [reply .travis.yml]: Sample travis configuration for build on .net core 2.2
https://github.com/mukmyash/QuartzHostedService/pull/4/files

Comment (mukmyash) [reply .travis.yml]: Sample travis configuration for build on .net core 2.2
https://github.com/mukmyash/QuartzHostedService/pull/4/files

Comment (maikebing) [reply .travis.yml]: Ok! is done.

**Review by mukmyash — APPROVED** [review]

**Pull request merged**